catalyst_voices/packages/catalyst_voices_models/lib/src/brands.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_assets/generated/colors.gen.dart';
+
+import 'package:flutter/material.dart';
+
+// [brands] is a Map that stores the ThemeData for all the
+// available brands in the Catalyst Voices app.
+// This map is used by the [BrandBloc] to properly
+// switch the active theme based on the BlocState.
```

Comment:
How about we use doc comments here: https://dart.dev/effective-dart/documentation#doc-comments. The one you used "double slash" is ignored when creating docs via dart doc